src/screens/OtherCountries.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {View,Text,StyleSheet,FlatList,ActivityIndicator,Dimensions} from 'react-native';
+import Drawer from '../components/Drawer';
+const screenwidth = Dimensions.get('window').width;
+
+export default class OtherCountries extends React.Component{
+    state = {
+        loading : true,
+        data : []
+    }
+    componentDidMount(){
+        this.fetchapi();
+    }
+    fetchapi = () => {
```

Comment:
To keep proper modularity, we have an api and hooks folder in the src. It's better you shift this logic there to maintain sync throughout the source code

Reply:
Ok will do that.